pkg/operators/ebpf/ebpf.go

```diff
@@ -543,6 +544,13 @@ func (i *ebpfInstance) tracePipe(gadgetCtx operators.GadgetContext) error {
 	return nil
 }
 
+// Golang representation of struct gadget_l3endpoint_t in include/gadget/types.h
+type ipAddr struct {
+	v6      [16]byte
+	version uint8
+	pad     [3]byte
```

Comment:
I think it's possible to use `_ [3]byte`

Reply:
ok, got it